### ISSUE TITLE
Update package version limitations

### ISSUE THREAD

**Issue by ClaraBuettner** [openego/eTraGo#488]
This should allow using python>3.8

**Comment by ClaraBuettner** [comment]
There are many deprecation warnings in the clustering when pandas is updated: 
```

/home/clara/etrago/eTraGo/etrago/cluster/networkclustering.py:162: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.
  no_elec_to_cluster = no_elec_to_cluster.append(new)
```

@CarlosEpia: Could you help me to change the code to avoid these warnings?

**Comment by CarlosEpia** [comment]
Of course. I will manage the deprecations problems related to Pandas and another one related to Shapely.

**Comment by CarlosEpia** [comment]
The deprecation warning for Pandas about the function append was solved in the branch features/#488-package-version-limits.
